src/classes/Country.cls

```diff
@@ -0,0 +1,117 @@
+public with sharing class Country {
+  //mapOfCountrywithCountryCode
+  Map<string,Map<string,Map<string,List<String>>>>  mapOfCountrywithCountryCode= new Map<string,Map<string,Map<string,List<String>>>>();
+  public Country(){
+    countryDetails();
+  }
+  boolean nf=true;
+  public  void countryDetails() {
+    //district
+    List<string> WestBengaldistrictname= new List<string>();
```

Comment:
Use camel case notation. Name should start with small letter. Fix it all the places